### ISSUE TITLE
Write test cases for interacting with each API endpoint

### ISSUE THREAD

**Issue by ryanhamley** [Skycatch/acopio-ui#9]
Right now there's a commented out test function in `App.js` that makes requests to the endpoints of the Acopio API. We should remove this and write test cases for interacting with each endpoint.

**Comment by sbaidon** [comment]
I can take care of this once CORS issues are dealt with

**Comment by ryanhamley** [comment]
Thanks! At the very least, we'd need the GET endpoints for v1.

**Comment by sbaidon** [comment]
will do

**Comment by ryanhamley** [comment]
@sbaidon The CORS issue was taken care of this morning if you want to take this on.

**Comment by sbaidon** [comment]
I have been trying to make the tests but some endpoints are not working yet

**Comment by ryanhamley** [comment]
@sbaidon Yeah, it seems like some of the database stuff is still being updated and changed around. Discussion is ongoing in Slack
